modules/stitching/src/matchers.cpp

```diff
@@ -768,7 +768,7 @@ void BestOf2NearestMatcher::match(const ImageFeatures &features1, const ImageFea
 
     // Set zero confidence to remove matches between too close images, as they don't provide
     // additional information anyway. The threshold was set experimentally.
-    matches_info.confidence = matches_info.confidence > 3. ? 0. : matches_info.confidence;
+    // matches_info.confidence = matches_info.confidence > 3. ? 0. : matches_info.confidence;
```

Comment:
What is about the comment above of this code?

Reply:
> Updated

Reply:
@alalek could you re-review/approve at earliest convenience? Thank you!

Reply:
Could you please share some dataset which shows the problem of this change? So we can take a look on the problem.

This code is here for eliminating of very close images (may happen on live capturing on mobile devices). Blending step with two close images provides unnecessary artifacts, so this threshold is applied here.

Ideally it would be great to provide dedicated parameter for this threshold (probably for the master branch, 3.4 is in the maintenance mode)